build/travis/job2_AppImage/build.sh

```diff
@@ -27,17 +29,31 @@ else
   makefile_overrides="" # use Makefile defaults
 fi
 
-# Build MuseScore AppImage inside Docker image
-docker run -i -v "${PWD}:/MuseScore" library/centos:6 \
-   /bin/bash -c "/MuseScore/build/Linux+BSD/portable/Recipe $makefile_overrides"
+# Build AppImage. Are we cross-compiling?
+case "$1" in
+  --32bit )
+    shift
+    # Build MuseScore AppImage inside 32-bit Docker image
+    docker run -i -v "${PWD}:/MuseScore" toopher/centos-i386:centos6 /bin/bash -c \
+      "linux32 --32bit i386 /MuseScore/build/Linux+BSD/portable/Recipe $makefile_overrides"
+    ;;
+  * )
+    [ "$1" == "--64bit" ] && shift || true
+    # Build MuseScore AppImage inside native (64-bit) Docker image
+    docker run -i -v "${PWD}:/MuseScore" library/centos:6 /bin/bash -c \
+      "/MuseScore/build/Linux+BSD/portable/Recipe $makefile_overrides"
+    ;;
+esac
```

Comment:
@probonopd In https://github.com/probonopd/AppImageKit/blob/master/README.md#objectives line 8 you write:<blockquote>Do not require recompilation. AppImages must be possible to create from already-existing binaries, without the need for recompilation. This greatly speeds up the AppImage creation process, since no compiler has to be involved. This also allows third parties to package closed-source applications as AppImages. (Nevertheless, it can be beneficial for upstream application developers to build from source specifically for the purpose of generating an AppImage.)</blockquote>
That leads me to think that yes, I can use crosscompiler to build musescore, and then just package the AppImage inside an ARM container (so gets the ARM libraries).